src/utils/generateTextEmailFromHTML/generateTextEmailFromHTML.ts

```diff
@@ -1,6 +1,69 @@
-import htmlToPlainText from 'textversionjs';
+import { convert } from 'html-to-text';
 
 export const generateTextEmailFromHTML = (html: string): string => {
-  const plainText = htmlToPlainText(html, { headingStyle: 'hashify' });
+  const plainText = convert(html, {
+    formatters: {
+      h1Formatter: function (elem, walk, builder, formatOptions) {
+        builder.openBlock({ leadingLineBreaks: formatOptions.leadingLineBreaks || 1 });
+        builder.addInline('# ');
+        walk(elem.children, builder);
+        builder.closeBlock({ trailingLineBreaks: formatOptions.trailingLineBreaks || 1 });
+      },
+      h2Formatter: function (elem, walk, builder, formatOptions) {
+        builder.openBlock({ leadingLineBreaks: formatOptions.leadingLineBreaks || 1 });
+        builder.addInline('## ');
+        walk(elem.children, builder);
+        builder.closeBlock({ trailingLineBreaks: formatOptions.trailingLineBreaks || 1 });
+      },
+      h3Formatter: function (elem, walk, builder, formatOptions) {
+        builder.openBlock({ leadingLineBreaks: formatOptions.leadingLineBreaks || 1 });
+        builder.addInline('### ');
+        walk(elem.children, builder);
+        builder.closeBlock({ trailingLineBreaks: formatOptions.trailingLineBreaks || 1 });
+      },
+      h4Formatter: function (elem, walk, builder, formatOptions) {
+        builder.openBlock({ leadingLineBreaks: formatOptions.leadingLineBreaks || 1 });
+        builder.addInline('#### ');
+        walk(elem.children, builder);
+        builder.closeBlock({ trailingLineBreaks: formatOptions.trailingLineBreaks || 1 });
+      },
+      h5Formatter: function (elem, walk, builder, formatOptions) {
+        builder.openBlock({ leadingLineBreaks: formatOptions.leadingLineBreaks || 1 });
+        builder.addInline('##### ');
+        walk(elem.children, builder);
+        builder.closeBlock({ trailingLineBreaks: formatOptions.trailingLineBreaks || 1 });
+      },
```

Comment:
```suggestion
      const formatter = (level) => (elem, walk, builder, formatOptions) => {
        builder.openBlock({ leadingLineBreaks: formatOptions.leadingLineBreaks || 1 });
        builder.addInline('#'*level); // inoythin style, do something similar here. you get the idea.
        walk(elem.children, builder);
        builder.closeBlock({ trailingLineBreaks: formatOptions.trailingLineBreaks || 1 });
      }
      .....
      h5Formatter: formatter(3),
```

Reply:
Created a separate `headingFormatter` function which accepts `level` and add "#" according to the heading type for formatting